src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -892,12 +893,19 @@ public virtual void remove_rollback_directory_if_exists(string packageName)
 
             var originalConfig = config;
 
+            var notFoundedMessage = new StringBuilder();

```

Comment:
We already have messaging on not found. Can you reuse that?

Reply:
Sorry I failed to find any message on not found except the one for `not found in source`, is that the one you are referring to? Otherwise, can you guide me where to find it? Thanks!